maxim/tests/test_crewai.py

```
@@ -0,0 +1,98 @@
+            delattr(Maxim, "_instance")
+
+        # Initialize Maxim and logger
+        self.maxim = Maxim({
+            "api_key": os.getenv("MAXIM_API_KEY"),
+            "base_url": os.getenv("MAXIM_BASE_URL"),
+            "debug": True,
+        })
+        self.logger = self.maxim.logger({"id": os.getenv("MAXIM_LOGGER_ID")})
+        
+        # Instrument CrewAI with Maxim logging
+        instrument_crewai(self.logger, True)
```

Ruff (0.12.2)

Boolean positional value in function call

(FBT003)